vm/src/cairo_run.rs

```diff
@@ -19,6 +19,7 @@ use arbitrary::{self, Arbitrary, Unstructured};
 
 #[cfg_attr(feature = "arbitrary", derive(Arbitrary))]
 pub struct CairoRunConfig<'a> {
+    #[cfg_attr(feature = "arbitrary", arbitrary(value = "main"))]
```

Comment:
sorry @Juan-M-V 
I don't get why we are adding this

Reply:
I wanted to just test the "main" entrypoint when generating `CairoRunConfig` with arbitrary. Otherwise most runs fail because they will look for a random string as their entrypoint